config.yml.example

```diff
@@ -1,6 +1,8 @@
 ---
 
-servers:
+environment: # optional
```

Comment:
This looks good to me.

Do you mind replacing `FOO_API` with the list of API keys [here](https://github.com/woodruffw/yossarian-bot#installation)? (With placeholder values, of couse).

Reply:
Done.